README.md

```diff
@@ -39,16 +37,16 @@ If you make changes to the website, repeat the steps to build the image and star
 
 ## Deploy preview
 
-Changes in the `dev` branch can be previewed and tested via the deploy preview site hosted by Netlify:
+Changes in the `dev` branch can be previewed and tested via the deploy preview site hosted by Cloudflare Pages:
 
-<https://dev--fluid-floeproject-org.netlify.app/>
+<https://dev.floeproject-org.pages.dev/>
```

Comment:
This dev preview deploy doesn't seem to be working. Has that been setup already or is the URL incorrect?

Reply:
I think this is the correct URL: https://floeproject-org.pages.dev/

Reply:
It won't until we push to that branch, which we haven't yet.